web/src/view.d.ts

```diff
@@ -0,0 +1,22 @@
+declare namespace OpenAPI2 {
+  export interface webviewView {
+    log?: string;
```

Comment:
this is the main thing to look at

separately, we should strategize on how to substitute these generated types for the handwritten types in types.ts

Reply:
Aww, it makes all of these optional fields? Why?

Reply:
in proto3, fields can only be optional. there's some good discussion here:
https://github.com/protocolbuffers/protobuf/issues/2497
https://capnproto.org/faq.html#how-do-i-make-a-field-required-like-in-protocol-buffers